Tests/MongoSwiftTests/SpecTestRunner/SpecTestUtils.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
```

Comment:
good idea to introduce one of these. the single `TestUtils` was bound to get bloated with spec test stuff eventually (if it wasn't already).